### PR TITLE
Undraft the bylaws: decision making [due May 17]

### PR DESCRIPTION
**Pull request by ebullient**
[🗳️ Vote progress](https://github.com/commonhaus/foundation/pull/158#issuecomment-2105941257 "IC_kwDOKRPTI859hh0J")

Let's undraft the Decision Making process.

Key revisions include clearer links to the 'Voting Rules' as defined in the CF Council documents and clear voting procedures to ensure all members can effectively participate.

voting group: @commonhaus/cf-egc 

Do one of the following:

- Approve the PR or react with 👍 (:+1:) if it looks good to you.
- Review with Comments or react with 👀 (:eyes:) if you're "ok" with it (it may not be your favorite).
- If you think it needs discussion or revision:
    - Use the +- button to make a suggestion.
    - Create a review, add your comments and require changes,

### CODE REVIEW

File: bylaws/5-decision-making.md [hunk 1]
```diff
@@ -37,34 +33,34 @@ Proposals typically begin as GitHub Discussions for collaborative refinement.
 
 ### Sense Vote
 
+Members express their stance on proposals using *reactions* to express their level of comfort or agreement with a proposal.
```
Comment (aalmiray) [inline bylaws/5-decision-making.md:36]: Express used twice in the same sentence 🤔

Comment (kenfinnigan) [inline bylaws/5-decision-making.md:36]:
```suggestion
Members express their stance on proposals using *reactions* to convey their level of comfort or agreement with a proposal.
```

File: bylaws/5-decision-making.md [hunk 2]
```diff
@@ -37,34 +33,34 @@ Proposals typically begin as GitHub Discussions for collaborative refinement.
...
 - **👀 (:eyes:):** Suggests acceptance or willingness to go along with the proposal, even if it's not the preferred choice.
 - **👎 (:-1:):** Signifies discomfort or disagreement with the proposal, requiring further discussion.
 
-Members are expected to *react* to express their level of comfort or agreement with a proposal,
-ensuring a comprehensive understanding of the community's stance.
-
 Discussion of the proposal should occur in comments on the PR or GitHub Discussion. If conversations happen elsewhere (in chat, for example), a summary of the conversation should be added as a comment if the discussion helped clarify the proposal or resolve a concern.
 
-If significant changes are needed, the current Discussion/PR should be closed as outdated, and replaced with a revised proposal.
+If significant revisions are necessary, the current proposal should be closed as outdated and replaced with a new proposal with revised content to maintain focus on the most current version and simplify navigation for members.
```
Comment (kenfinnigan) [inline bylaws/5-decision-making.md:46]:
```suggestion
If significant revisions are necessary, the current proposal should be closed as outdated and replaced with a new proposal.
The new proposal has revised content to maintain focus on the most current version and simplify navigation for members.
```

**Review by kenfinnigan — APPROVED** [review]
Couple of minor suggestions

File: bylaws/5-decision-making.md (hunk 1 above)
Comment (ebullient) [reply bylaws/5-decision-making.md:36]: image

Don't just tell me it's wrong, help fix it next time. Faster that way.

File: bylaws/5-decision-making.md [hunk 3]
```diff
@@ -37,34 +33,35 @@ Proposals typically begin as GitHub Discussions for collaborative refinement.
...
-
-- Arrange a meeting with all relevant parties to discuss the proposal.
-- Use a structured, time-boxed format to address objections.
-- Conclude with a vote to decide whether to move forward despite unresolved objections.
+If consensus can not be reached:
 
-Record the meeting's outcome in the original Discussion/PR to maintain transparency and traceability.
+- A member of the CFC will arrange a meeting with all relevant parties to discuss the proposal.
+- The meeting will use a structured, time-boxed format to allow the proposal and objections to be presented and discussed.
+- The meeting will conclude with a vote, subject to [voting rules][cfc-sv], to decide whether to proceed despite unresolved objections.
```
Comment (sigrunixia) [inline bylaws/5-decision-making.md:63]: Reading out loud, it doesn't flow right. Switching from:

`- The meeting will conclude with a vote, subject to [voting rules][cfc-sv], to decide whether to proceed despite unresolved objections.`

To: 
`- The meeting will conclude with a vote, subject to [voting rules][cfc-sv], that will decide whether to proceed despite unresolved objections.`

File: bylaws/5-decision-making.md (hunk 1 above)
Comment (aalmiray) [reply bylaws/5-decision-making.md]: I would if I knew how to fix it. I just know it looks weird. 

Me no speako English very good 😅

**Review by aalmiray — APPROVED** [review]

**Review by cowtowncoder — APPROVED** [review]

File: bylaws/5-decision-making.md [hunk 4]
```diff
@@ -37,34 +33,35 @@ Proposals typically begin as GitHub Discussions for collaborative refinement.
 
 ### Sense Vote
 
+Members express their their level of comfort or agreement with proposals using *reactions*.
+
+Sense votes typically use the following reactions, though alternatives may be suggested for specific proposals:
```
Comment (evanchooly) [inline bylaws/5-decision-making.md:38]: Sense votes?

File: bylaws/5-decision-making.md [hunk 5]
```diff
@@ -37,34 +33,35 @@ Proposals typically begin as GitHub Discussions for collaborative refinement.
...
 - **👎 (:-1:):** Signifies discomfort or disagreement with the proposal, requiring further discussion.
 
-Members are expected to *react* to express their level of comfort or agreement with a proposal,
-ensuring a comprehensive understanding of the community's stance.
-
 Discussion of the proposal should occur in comments on the PR or GitHub Discussion. If conversations happen elsewhere (in chat, for example), a summary of the conversation should be added as a comment if the discussion helped clarify the proposal or resolve a concern.
 
-If significant changes are needed, the current Discussion/PR should be closed as outdated, and replaced with a revised proposal.
+If significant revisions are necessary, the current proposal should be closed as outdated and replaced with a new proposal.
+The new proposal has revised content to maintain focus on the most current version and simplify navigation for members.
```
Comment (evanchooly) [inline bylaws/5-decision-making.md:47]: I love this.

File: bylaws/5-decision-making.md [hunk 6]
```diff
@@ -37,34 +33,35 @@ Proposals typically begin as GitHub Discussions for collaborative refinement.
...
-
-- Arrange a meeting with all relevant parties to discuss the proposal.
-- Use a structured, time-boxed format to address objections.
-- Conclude with a vote to decide whether to move forward despite unresolved objections.
+If consensus can not be reached:
 
-Record the meeting's outcome in the original Discussion/PR to maintain transparency and traceability.
+- A member of the CFC will arrange a meeting with all relevant parties to discuss the proposal.
+- The meeting will use a structured, time-boxed format to allow the proposal and objections to be presented and discussed.
+- The meeting will conclude with a vote, subject to [voting rules][cfc-sv], that will decide whether to proceed despite unresolved objections..
```
Comment (evanchooly) [inline bylaws/5-decision-making.md:63]: this makes me slightly nervous but is probably necessary to avoid procedural logjams.

**Review by evanchooly — APPROVED** [review]

File: bylaws/5-decision-making.md (hunk 4 above)
Comment (ebullient) [reply bylaws/5-decision-making.md:38]: That is terminology from Martha's rules. It is not (strictly speaking) a formal vote. But if there is consensus, you can move forward.

File: bylaws/5-decision-making.md (hunk 6 above)
Comment (ebullient) [reply bylaws/5-decision-making.md:63]: This is the final formal vote part (as distinct from the sense vote above). 

This is how you resolve things when consensus can't be reached (a real, formal vote). In my mind, it's the last resort, and getting unstuck is its purpose.

Comment (gavinking) [inline bylaws/5-decision-making.md:63]:
```suggestion
- The meeting will conclude with a vote, subject to [voting rules][cfc-sv], that will decide whether to proceed despite unresolved objections.
```

**Review by cealsair — APPROVED** [review]

**Review by gavinking — APPROVED** [review]

**Review by maxandersen — APPROVED** [review]

**Pull request merged**